lib/web_ui/test/paragraph_test.dart

```diff
@@ -186,4 +186,70 @@ void main() async {
     TextMeasurementService.clearCache();
     TextMeasurementService.enableExperimentalCanvasImplementation = false;
   });
+
+  testEachMeasurement('getLineBoundary (single-line)', () {
+    final ParagraphBuilder builder = ParagraphBuilder(ParagraphStyle(
+      fontFamily: 'Ahem',
+      fontStyle: FontStyle.normal,
+      fontWeight: FontWeight.normal,
+      fontSize: 10,
+    ));
+    builder.addText('One single line');
+    final Paragraph paragraph = builder.build();
+    paragraph.layout(const ParagraphConstraints(width: 400.0));
+
+    // "One single line".length == 15
+    for (int i = 0; i < 15; i++) {
+      expect(
```

Comment:
Add a `reason:` to this expect that tells what `i` is, so that it's easier to debug.

For example:
`reason: 'failed at offset $i'`

Reply:
Nice pro-tip! Thanks I'll add it.